org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/interpreter/impl/XbaseInterpreter.java

```diff
@@ -115,7 +115,7 @@
 public class XbaseInterpreter implements IExpressionInterpreter {
 	
 	@SuppressWarnings("serial")
-	static class ReturnValue extends RuntimeException {
+	public static class ReturnValue extends RuntimeException {
```

Comment:
Can you please add a `@Since` tag for documentation purpose. If you don't mind, this class would now also deserve a brief JavaDoc comment to clarfiy its purpose.

Reply:
since tag has been added.

Reply:
JavaDoc comment has been added.